docs/api/js/accounts.md

```diff
@@ -111,7 +110,7 @@ const wallet = unconnectedWallet.connect(provider).connectToL1(ethProvider);
 ### Getting the zkSync L1 smart contract
 
 ```typescript
-async getMainContract(): Promise<Contract>
+async Adapter.getMainContract(): Promise<IZkSync>
```

Comment:
Looks weird, what is Adapter?

Reply:
it is ts class..

Reply:
We have neither import nor deffinition. 
From the user's perspective, it's just a random word, which they should find by search or where? 
How they can initialize this class? is it ethers class or zksync?